tests/_test_mscolab/test_server.py

```diff
@@ -234,19 +235,27 @@ def test_get_all_changes(self):
             operation, token = self._create_operation(test_client, self.userdata)
             fm, user = self._save_content(operation, self.userdata)
             fm.save_file(operation.id, "content2", user)
+            all_changes = fm.get_all_changes(operation.id, user)
             response = test_client.get('/get_all_changes', data={"token": token,
                                                                  "op_id": operation.id})
             assert response.status_code == 200
             data = json.loads(response.data.decode('utf-8'))
+            time.sleep(1)
             assert len(data["changes"]) == 2
+            assert all_changes[0]["id"] == 2
+            assert all_changes[0]["id"] > all_changes[1]["id"]
+            assert all_changes[0]["created_at"] > all_changes[1]["created_at"]
 
     def test_get_change_content(self):
         assert add_user(self.userdata[0], self.userdata[1], self.userdata[2])
         with self.app.test_client() as test_client:
             operation, token = self._create_operation(test_client, self.userdata)
             fm, user = self._save_content(operation, self.userdata)
+            # we need to wait to get an updated created_at
+            time.sleep(1)
             fm.save_file(operation.id, "content2", user)
             all_changes = fm.get_all_changes(operation.id, user)
+            # the newest change is on index 0, because it has a recent created_at time
```

Comment:
This comment is in the wrong place now, it belongs to the new asserts in the other test.